packages/subgraph/subgraph.yaml.js

```diff
@@ -1,18 +1,32 @@
+const fs = require("fs");
+
+const { addresses } = require("@liquity/lib-ethers/deployments/mainnet.json");
+// const { addresses } = require("@liquity/lib-ethers/deployments/dev.json");
```

Comment:
It would be nice to be able to pass this as a param, or have something like `prepare-local` and `prepare`, or whatever, so we don’t have to manually uncomment this.

Reply:
Done 👍

Reply:
Awesome, thanks!